scripts/ci/_utils.sh

```diff
@@ -552,6 +561,85 @@ function set_current_image_variables {
     fi
 }
 
+function build_image_manifest() {
+    verbose_docker inspect "${AIRFLOW_IMAGE}" > "manifests/${AIRFLOW_BASE_TAG}.json"
+    verbose_docker build \
+    --build-arg AIRFLOW_BASE_TAG="${AIRFLOW_BASE_TAG}" \
+    --tag="${AIRFLOW_LOCAL_MANIFEST_IMAGE}" \
+    -f- . <<EOF
+ARG AIRFLOW_BASE_TAG
+FROM scratch
+
+COPY "manifests/${AIRFLOW_BASE_TAG}.json" .
+
+CMD ""
+EOF
+}
+
+function get_local_image_info() {
+    TMP_MANIFEST_LOCAL_JSON=$(mktemp)
+    TMP_MANIFEST_LOCAL_SHA=$(mktemp)
+    set +e
+    verbose_docker rm --force "local-airflow-manifest"  >/dev/null 2>&1
+    if ! verbose_docker create --name "local-airflow-manifest" \
+        "${AIRFLOW_LOCAL_MANIFEST_IMAGE}"  >/dev/null 2>&1 ; then
+        echo
+        echo "Local docker not available"
```

Comment:
```suggestion
        echo "Local airflow CI docker image not available"
```

Reply:
Or similar - "image" being the key bit to add here.

Reply:
> Can you add some docs/comments in the PR somewhere about
> 
> 1. what this is doing -- I'm having trouble following the logic; and
> 2. why (your comment in the PR should go in the script somewhere)

Good point. I added comments describing both.

Reply:
Added full description + a bit nicer output showing pull progress as well as step of the docker build during pre-commit.